### ISSUE TITLE
Script cannot locate the jar file

### ISSUE THREAD

**Issue by Rmarsh92** [colebob9/AutoChunky#3]
Getting this error
![screen shot 2016-08-02 at 11 25 13](https://cloud.githubusercontent.com/assets/18497875/17325493/ced763da-58a3-11e6-855d-874169e9ab26.png)
![screen shot 2016-08-02 at 11 24 38](https://cloud.githubusercontent.com/assets/18497875/17325500/d086581c-58a3-11e6-966e-aef77e8d7d19.png)

As you can see, the jar is in there.

**Comment by chlorinezorine** [comment]
Alright, probably fixed in my develop branch (over on commit 5747c75). It's a Mac related problem, so test it out and get back to me if it works (as I don't have a Mac myself).

I have attached the fixed version in this comment as well for convenience.
[AutoChunky Mac Dev 1.zip](https://github.com/colebob9/AutoChunky/files/397469/AutoChunky.Mac.Dev.1.zip)

**Comment by chlorinezorine** [comment]
Little update: I seem to have the same exact error on Ubuntu Linux (16.04). My addition fixed the error on that. I'll still wait on if it works on a Mac before closing this issue.

**Comment by Rmarsh92** [comment]
![screen shot 2016-08-03 at 09 33 41](https://cloud.githubusercontent.com/assets/18497875/17359203/7f6c93f0-595d-11e6-92d2-cd89d07e4c3e.png)
Its now throwing this. I'll also add that i'm having the same issue on my WIN10 machine

**Comment by Rmarsh92** [comment]
Whoops forgot a line
![screen shot 2016-08-03 at 09 41 57](https://cloud.githubusercontent.com/assets/18497875/17359472/999cd680-595e-11e6-8a10-bb24b37e2703.png)

**Comment by chlorinezorine** [comment]
It looks like all the names of the scenes have slashes in them. It's tripping up shlex. ~~I'll be researching how to read those names correctly, but for right now,~~ try without the slashes in the names.

**Comment by chlorinezorine** [comment]
~~Odd, I can't seem to replicate it on my end.~~ Mind making a pastebin (or GitHub Gist) for both the `queue.txt` and the error it's throwing?

Edit: I happened to replicate it with typing out the exact name of the last render in that list. I also noticed the names had font names in them. Did you happen to make the `queue.txt` with a rich text editor like Microsoft Word instead of a plain text one like Notepad++? My guess is you used TextEdit on your Mac and forgot [to format it as plain text.](http://superuser.com/a/12600)
